### ISSUE TITLE
Gecko intouch2 Whirlpoolcontrol

### ISSUE THREAD

**Issue by Konni1234** [ioBroker/AdapterRequests#720]
Adapter for Gecko Alliance Whilpoolcontrol.
https://geckoportal.com/

A python lib is aviable:
https://github.com/gazoodle/geckolib

Also an sample integrtion for Home Assistant
https://github.com/gazoodle/gecko-home-assistant

**Comment by HJS72** [comment]
Any news about this request? @Konni1234: Did you manage to control the Gecko Whirlpool?

**Comment by Telefisch** [comment]
not decided which pool I should buy but I‘m interrested in this adapter also

**Comment by rrov1** [comment]
Hi, I also own 2 gecko in.touch2 devices. Currently I use geckolib an some Javascript/Python scripts to read states and manage some device properties (check out: https://github.com/rrov1/SwimSpa and [Steuerung Gecko in Touch 2 per iob möglich? (Wellis)](https://forum.iobroker.net/topic/58779/steuerung-gecko-in-touch-2-per-iob-m%C3%B6glich-wellis)).

Perhaps later this year I will decide if it possible for me to create an adapter based on gazoodle's geckolib, but it seems this kind of device is not very often in use.

rrov1

**Comment by OktaMen** [comment]
This would be super great.
Then i could optimate the electric consumption with my Solar Panels.

**Comment by rrov1** [comment]
> This would be super great. Then i could optimate the electric consumption with my Solar Panels.
Could you open a new issue in: https://github.com/rrov1/SwimSpa/issues, so that we can discuss this (german is also ok there)? I can share my current solution with you (it's not a adpater, only a javascript, but it works).

**Comment by mcm1957** [comment]
> This would be super great. Then i could optimate the electric consumption with my Solar Panels.

Pleas use thumbs up at THE FIRST POST to show interrest. me too posts do not raise visibility. (Sorting is done by thumbs up from time to time)

**Comment by thomasklein1975** [comment]
additional to exisiting funtionalities it would be great if the circulating pump could be controlled. To manage color/zones for leds while using modul "in mix" would be great too.

**Comment by thogens** [comment]
Any news here? Would be so great to get an adapter!
Thomas

**Comment by rrov1** [comment]
> Any news here? Would be so great to get an adapter! Thomas

No. I'm currently fine with my script solution, it works nearly like an adapter when it's set up.
rrov1
